opennlp-docs/src/docbkx/namefinder.xml

```diff
@@ -506,8 +506,8 @@ System.out.println(result.toString());]]>
 			To perform cross validation the ObjectStream must be resettable.
 			<programlisting language="java">
 				<![CDATA[
-FileInputStream sampleDataIn = new FileInputStream("en-ner-person.train");
-ObjectStream<NameSample> sampleStream = new PlainTextByLineStream(sampleDataIn.getChannel(), StandardCharsets.UTF_8);
+InputStreamFactory dataIn = new MarkableFileInputStreamFactory(new File("en-ner-person.train"));
+ObjectStream<NameSample> sampleStream = new PlainTextByLineStream(dataIn, StandardCharsets.UTF_8);
```

Comment:
I think `PlainTextByLineStream` is (at least on `master`) an `ObjectStream<String>`. So the code with `NameSample` as generic type doesn't compile?